app/models/card.rb

```diff
@@ -1,12 +1,17 @@
 class Card < ActiveRecord::Base
 
-  validates :original_text, presence: true
+  validates :original_text, :translated_text, :review_date,   presence: true
+  validate  :check_duplication
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.